aten/src/ATen/EmptyTensor.cpp

```diff
@@ -98,6 +98,37 @@ size_t computeStorageNbytes(
 #endif
 }
 
+// not including mobile-only macros in this function,
+// since mobile shouldn't be using symints.
+SymInt computeStorageNbytesSymInt(
+    SymIntArrayRef sizes,
+    SymIntArrayRef strides,
+    SymInt itemsize_bytes,
+    size_t storage_offset
+  ) {
+  // size of the underlying storage is 1 bigger than the offset
+  // of the last element according to stride
+  SymInt size = storage_offset + 1;
+  for (const auto i : c10::irange(sizes.size())) {
+    // Note: the non-symint version of this code includes overflow checks.
+    // The code that eventually materialized the SymInt will need to be
+    // in charge of handling overflow.
+    // TODO: this should be (sizes[i] - 1), but that breaks, figure out why.
+    auto tmp1 = sizes[i];
+    auto tmp2 = strides[i];
+    SymInt next_size = tmp1 * tmp2;
```

Comment:
@Chillee this line is giving me problems locally - do you have any idea of what's going on? Error:
```
RuntimeError: NotImplementedError: argument of type: <class 'torch.fx.experimental.symbolic_shapes.PySymInt'>
```

Reply:
Still digging, but I have a feeling it has to do with the way `stride()` is meant to be implemented in `FakeTensor` (maybe I did it wrong?) It's in the previous PR here: https://github.com/pytorch/pytorch/pull/81237/files#diff-39e82af71afdadbc56a4ecf552ec668ddcda794f8ea3ec41edaef23456fd56e9R474